mautic/mautic
# [UX] Sending at, Activate at and Available from (statuses)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR aligns the schedule options with the status label used by a bundle. Example: Since you make an Asset available for use, it makes sense to schedule "Available from" and "Unavailable from" instead of the generic "Activate at/Deactivate at".

All changes:
- Fixes the broken schedule logic for Emails. When you create a Template Text Message (sms), the "Publish/Activate at" options are hidden. They should also be hidden for Template Emails.
- Aligns Segment Emails and SMS to use "Start sending at" since this way better expresses the functionality 
- Uses "Available from" instead of "Activate at" for resources where the publish label is "Available for use" to improve consistency and feedback (users create a better relationship between the fields)






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Plugins > Twilio > Fill anything and set Active > Save
3. Open Components > Emails > New > Template > See any "Start sending at" options > Save
4. Repeat the step 3 for Template Text Messages (SMS)
5. Open Components > Emails > New > Segment > See "Start sending at" options  > Save
6. Repeat the step 5 for Segment Text Messages (SMS)
7. Open Components > Emails > Open the created Template Email > Dropdown > Add A/B test > See no "Start sending at" options
8. Open Components > Emails > Open the created Segment Email > Dropdown > Add A/B test > See "Start sending at" options
9. Open Components > Assets > New > See "Available from" options
10. Repeat the step 9 for Marketing Messages, Forms and Landing pages
